app/api/v_sphere/virtual_machine.rb

```diff
@@ -261,6 +270,66 @@ def status
       end
     end
 
+    def users
+      GitHelper.open_repository PuppetParserHelper.puppet_script_path do
+        users = PuppetParserHelper.read_node_file(name)
+        (users['admins'] + users['users']).uniq
+      end
+    end
+
+    def commit_message(git_writer)
+      if git_writer.added?
+        'Add ' + name
+      elsif git_writer.updated?
+        'Update ' + name
+      else
+        ''
+      end
+    end
+
+    def set_users(ids)
+      begin
+        GitHelper.open_repository(PuppetParserHelper.puppet_script_path) do |git_writer|
+          all_users = PuppetParserHelper.read_node_file(name)
+          sudo_users = all_users['admins']
+          new_users = User.find_all_by_id(ids)
+          puppetscript = Puppetscript.node_script(name, sudo_users, new_users)
+          git_writer.write_file(name, puppetscript)
+          message = commit_message(git_writer)
+          git_writer.save(message)
+        end
+      rescue Git::GitExecuteError
+        { alert: 'Could not push to git. Please check that your ssh key and environment variables are set.' }
```

Comment:
It's a bit pointless to return the error if you're not actually displaying it.
If you don't want to display it, maybe use Rails.logger.info to write the error in the logs.

Reply:
this is handeled by #357